src/components/interpretations/InterpretationsPanel.js

```diff
@@ -68,15 +68,17 @@ class InterpretationsPanel extends Component {
                 anchor="right"
                 classes={{ paper: classes.panel }}
             >
-                <Interpretations
-                    d2={this.context.d2}
-                    id={mapId}
-                    type="map"
-                    currentInterpretationId={interpretationId}
-                    onCurrentInterpretationChange={
-                        this.onCurrentInterpretationChange
-                    }
-                />
+                <div>
```

Comment:
Why is this extra DOM node required?

Reply:
Without the wrapper DOM, the Drawer does not produce a scrollbar after changing the root interpretation component div to be flex container. I had to wrap it inside an extra div in order for it to overflow with a scrollbar. I've tried to add/modify the Drawer's css rules but the scrollbar does not appear when the component overflows vertically.

Reply:
Hmm... would it be possible to move that wrapper div into the `<Interpretations>` component then?  It's not obvious to a consumer of the component that they need to wrap it in a `<div>`.  If it's too much effort to rework the component now let's add a GitHub or JIRA issue to make sure we clean it up later.

Reply:
Yeah, I moved the wrapper to the component itself and updated the dependencies 👍